packages/alfa-rules/test/sia-r6/rule.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+
+import R6, { Outcomes } from "../../src/sia-r6/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test("evaluate() passes an html element which has identical primary language subtags for its lang and xml:lang attributes.", async (t) => {
+    const target = <html lang="en" xml:lang="en"></html>;
+  
+    const document = Document.of([target]);
+  
+    t.deepEqual(await evaluate(R6, { document }), [
+      passed(R6, target, {
+        1: Outcomes.HasMatchingLanguages,
+      }),
+    ]);
+  });
+
+  test("evaluate() passes an html element which has identical primary language subtags for its lang and xml:lang attributes. The extended language subtags also match.", async (t) => {
+    const target = <html lang="en-GB" xml:lang="en-GB"></html>;
+  
```

Comment:
```suggestion
  test("evaluate() passes an html element which has identical primary language subtags for its lang and xml:lang attributes. The extended language subtags don't match.", async (t) => {
    const target = <html lang="en-GB" xml:lang="en-US"></html>;
  
```
That's a case where ACT rules want to show "good code" in its passing example (so perfect match between the attributes), but we want to test the border conditions of the code.
The rule is only checking for the primary tag, not the region subtag. To make sure this is correctly done, we need a case where only the primary tags match.